client/src/components/tools/OnlineList.jsx

```diff
@@ -69,15 +76,28 @@ class OnlineList extends React.PureComponent {
   }
 
   render() {
-    const { status, players, online, loading } = this.state;
+    const { active, status, players, online, loading } = this.state;
     return (
       <Segment className="gm_tools-container">
         <Segment>
           <Icon name="power off" color={status === true ? 'green' : 'red'} /> Eden is {status === true ? 'online' : 'offline'}
         </Segment>
 
         <Segment.Group className="gm_online-count" raised>
-          <Segment>{online} Characters Online</Segment>
+          <Segment>
+            {online} characters online
+            {active != null && (
```

Comment:
This will only display if the query completes successfully. Otherwise it'll default to the old UI